README.md

```diff
@@ -1437,6 +1459,15 @@ jc.find({ type: 'jobType', status: 'ready' })
   });
 ```
 
+Non-Meteor worker scripts cannot use the `jc.find(...).observe(...)` portion of the above example, but they can achieve the same effect via the [DDP](https://www.npmjs.com/package/ddp) package's `observe` functionality. Using an established DDP connection named, for instance, `ddp`, we can replace the snippet with
```

Comment:
Hi, this is largely already covered in the documentation for `q.trigger()` in the npm package, here: https://www.npmjs.com/package/meteor-job#q-trigger

I've been trying (perhaps somewhat unsuccessfully) to keep all of the "plain node.js worker" details in the npm package docs, since the majority of job-collection users will probably never write a plain node.js worker.